### DISCUSSION TITLE
Icons in the header-bar look out of place

### DISCUSSION THREAD

**Discussion by marcinjahn** [elementary/mail#525] [General]
The icons in the header-bar of Geary look outdated to me. This might be just a personal taste, maybe not. To me they just seem "unprofessional".

![image](https://user-images.githubusercontent.com/10273406/102473500-cf594a80-4057-11eb-993d-1c54738f0c6e.png)

**Comment by danirabbit** [comment]
@Loreno10 can you elaborate on what you mean? What specifically makes the icons feel "outdated" or "unprofessional"? Can you provide examples of icons that you feel are "modern" and "professional" and articulate the differences between the two?

**Comment by marcinjahn** [comment]
I’ll try to explain what I mean.

1. The icons look huge, and very distracting. Also the “strong” colors have an impact here. The thing is - in the Mail app, the main focus should be put to the content - mails list and the selected mail. The icons of the app should be much more "quiet". This is why Mail in macOS has them done in wire-frame style - because these icons/options are not the main focus of the app. Now, if we look at another example - System Settings app. There, the icons are big and colorful (both eOS and macOS) - that's because this is the content of the app, there is nothing else to show! I think that this is one of the things that is done wrong in eOS. Icons are always designed as if THEY were the main focus - that's not always the case.
2. There is no consistency in the icons’ style. Some of them are “3D”, or rather projected from some angle (box). Some others are presented more “flat” (letter, pencil on a piece of paper).
3. They are not “leveled” properly - the flag makes an impression as if it is a bit higher than the rest, because its “mass” is at the top. The bottom part of this icon is very thin.
4. Generally, the choice of colors is very “cartoonish”, they are a bit unpleasant to look at. This remark is about most of the icons in eOS. The colors are too flat, they lack some "depth". In general, I don't really like Gnome icons (Sorry, I also wanted to explain what I meant by "outdated". The eOS icons are not far from https://commons.wikimedia.org/wiki/GNOME_Desktop_icons#:~:text=GNOME%20Desktop%20icons%20contains%20the,files%2C%20directories%2C%20and%20devices), and eOS is very similar to them

Regarding the examples. Sorry for no surprises: macOS. However, I have to say that also MS is currently on a good path. Their current effort with update of the icons throughout their services is quite good (although they have so many apps/services, that it’ll take them ages to update all of them). I really like their Office icons on macOS (much better than in the Windows version of Office).
Regarding Fluent iteself - just have a look. Isn't that a very well designed set of icons?

![image](https://user-images.githubusercontent.com/10273406/102538419-87184780-40ac-11eb-8613-d11b5e2c2649.png)

Unfortunately I do not have a direct comparison between a (in my opinion) well-designed icon set and eOS icon set. However, I have an idea:  have a look at eOS System Settings app (main grid) and the macOS Settings (preferably Catalina, Big Sur is worse in icons design I think).
Both apps are very similar, it’s clear where eOS took inspiration from (and it’s totally ok!). Just look at them both and answer honestly - which icon set has more "heart"?
I know that you guys are probably very attached to your own icon set, so for sure you will find a lot of "heart" in your version of the icons, but it's just an idea :)

Again, this is just my opinion, and maybe I’m not right. I just have a general feeling that eOS really lacks good icons. Not only in the Mail app, but throughout the system. Sorry for generalizing the issue outside of the Mail app.

One last thing - my complaints are only because I think that elementaryOS is a special thing in the Linux world. I love it, and I wish I could help in its evolution in some way, even by sharing my opinion!

**Comment by jlnr** [comment]
I really liked macOS app icons before Big Sur, what set them apart from Microsoft's and elementary's is that they had more interesting shapes and angles (fewer axis-aligned rectangles, more depth).

However, in general I'm not a fan of how Apple removed colors from icons (or in Big Sur, shapes). I think that "deference to the content" is mostly a rationalization for monochrome designs that look "cleaner" and more consistent at a first glance, but are slower to use because everything looks the same. Acting on emails is not a use case that needs to be de-emphasized. Some colors may stand out too much, especially in the dark theme, but a black & white outline design would be a step back for usability, in my opinion.
